trigger/task.go

```diff
@@ -761,7 +761,7 @@ func (t *taskTriggers) taskRuntimeChange(sub *event.Subscription) (*notification
 }
 
 func isFailedTaskStatus(status string) bool {
-	return status == evergreen.TaskFailed || status == evergreen.TaskSystemFailed || status == evergreen.TaskTestTimedOut
+	return utility.StringSliceContains(evergreen.TaskFailureStatuses, status)
```

Comment:
This is the same as `evergreen.IsFailedTaskStatus`, so this function could just be removed entirely.

Reply:
Is there any reason why this check originally omitted other failure statuses like system unresponsive and setup failed?

Reply:
There is no reason that I know of.

Reply:
My guess is that the statuses that weren't here (for example TaskSetupFailed,TaskSystemUnresponse,TaskSystemTimedOut) are outdated (I know system unresponse isn't used anymore since we use details but I'd have to do some digging to determine if the others are also outdated), but it also might be that we purposefully choose not to trigger on these things to avoid noise. If it doesn't seem like those statuses are used much then it should be fine to make this change.

Reply:
I defer to Annie. These statuses are too common for me to risk it. I have reverted my change.